tests/api.c

```diff
@@ -10369,6 +10514,28 @@ static void test_wolfSSL_set_options(void)
     #endif /* defined(OPENSSL_EXTRA) && !defined(NO_CERTS) && \
              !defined(NO_FILESYSTEM) && !defined(NO_RSA) */
 }
+	  #if defined(OPENSSL_EXTRA) && !defined(NO_CERTS)
+static int verify_cb(int ok, X509_STORE_CTX *ctx)
+{
+	  (void) ok;
+		(void) ctx;
+		printf("ENTER verify_cb\n");
+		return SSL_SUCCESS;
+}
+#endif
+
+static void test_wolfSSL_X509_STORE_CTX(void)
+{
+	  #if defined(OPENSSL_EXTRA) && !defined(NO_CERTS)
+	  X509_STORE_CTX *ctx   = NULL ;
+
+    printf(testingFmt, "test_wolfSSL_X509_STORE_CTX(()");
+		AssertNotNull(ctx = X509_STORE_CTX_new());
```

Comment:
Can you fix spacing here?  Looks like there might be a few hard tabs.

Reply:
FIxed